---
depth_lib/Object object
geometry_msgs/PointStamped[4] endPoints
